### PR TITLE
minFraud API Documentation (request) content

### PR DESCRIPTION
**Pull request by wesrice**

### CODE REVIEW

File: content/minfraud/_api-documentation/examples.mdx [hunk 1]
```diff
@@ -0,0 +1,217 @@
+export const request = `
```
Comment (kevcenteno) [inline content/minfraud/_api-documentation/examples.mdx:1]: Where is this file used?

File: content/minfraud/_includes/request.json [hunk 2]
```diff
@@ -0,0 +1,201 @@
+{
```
Comment (kevcenteno) [inline content/minfraud/_includes/request.json:1]: Where is this used?  Also, the file is called `request.json`, but this contains response data

File: content/minfraud/_includes/score-request.mdx [hunk 3]
```diff
@@ -0,0 +1,3 @@
+curl -d \
```
Comment (kevcenteno) [inline content/minfraud/_includes/score-request.mdx:1]: Where is this file used?

File: src/components/Curl/Curl.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+            styles.response,
+          )}
+          showLineNumbers={false}
+        >
+          <code
+            className="language-json"
+          >
+            {`< HTTP/2 200
+< content-type: text/html; charset=utf-8
+< set-cookie: __cfduid=d5180fc96c14bfdf800067d787a77baae1613692378; expires=Sat, 20-Mar-21 23:52:58 GMT; path=/; domain=.wesrice.com; HttpOnly; SameSite=Lax
```
Comment (kevcenteno) [inline src/components/Curl/Curl.tsx:51]: We should remove the `set-cookie`, `x-*`, and `cf-ray` headers from this

File: content/minfraud/_api-documentation/request-object-reference.mdx [hunk 5]
```diff
@@ -0,0 +1,962 @@
...
+      'Min': 0,
+      'Max': '1e14-1',
+    }}
+    type="decimal"
+  >
+    The per-unit price of this item in the shopping cart. This should use the
+    same currency as the order currency.
+  </Property>
+
+  <Footnote>
```
Comment (kevcenteno) [inline content/minfraud/_api-documentation/request-object-reference.mdx:895]: `<Footnote>` is isn't rendering properly:

image

**Review by kevcenteno — CHANGES_REQUESTED** [review]
I don't envy you for writing all this content, but well done.  A few issues though and a question: why are there empty files in this PR?

File: content/minfraud/_api-documentation/request-object-reference.mdx (hunk 5 above)
Comment (wesrice) [reply content/minfraud/_api-documentation/request-object-reference.mdx]: I pushed a fix for this.

File: src/components/Curl/Curl.tsx (hunk 4 above)
Comment (wesrice) [reply src/components/Curl/Curl.tsx]: This component was an idea I was playing with. I think it'd be nice to have a component for request/response examples that stacks two codeblocks – a code example for a curl command w/ the json body and then another for the response.

I've saved this in another branch and might bring it back in the response content pr.

File: content/minfraud/_includes/score-request.mdx (hunk 3 above)
Comment (wesrice) [reply content/minfraud/_includes/score-request.mdx]: This file has been removed.

File: content/minfraud/_includes/request.json (hunk 2 above)
Comment (wesrice) [reply content/minfraud/_includes/request.json]: This file has been remove

File: content/minfraud/_api-documentation/examples.mdx (hunk 1 above)
Comment (wesrice) [reply content/minfraud/_api-documentation/examples.mdx]: This file has been removed.

**Comment by wesrice** [comment]
@kevcenteno Aight. I think this is ready for another look.

**Review by kevcenteno — CHANGES_REQUESTED** [review]
I found one issue on the API documentation page:  The spacing between the page heading and the content is much larger than the guide pages.
image

**Comment by wesrice** [comment]
@kevcenteno I pushed a fix for this.

File: src/components/Schema/Schema.tsx [hunk 6]
```diff
@@ -60,6 +68,66 @@ const Schema: React.FC<ISchema> = (props) => {
...
+      (accumulator: any, child) => {
+        if (child?.props?.name && child?.props?.example) {
+          const { example, name, type } = child.props;
+
+          let formattedExample = example;
+
+          if (type === 'object' || type.startsWith('array')) {
+            formattedExample = JSON.parse(formattedExample);
+
+            console.log(formattedExample);
```
Comment (kevcenteno) [inline src/components/Schema/Schema.tsx:82]: This can be removed.

**Review by kevcenteno — CHANGES_REQUESTED** [review]
One last thing and we're good for sure

File: src/components/Schema/Schema.tsx (hunk 6 above)
Comment (wesrice) [reply src/components/Schema/Schema.tsx]: Good catch. Removed.

**Pull request merged**